buddyboss/buddyboss-platform
# PROD-683 #3179 - Fix: the [bbp-search] shortcode is not showing the search input

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3179 .

### How to test the changes in this Pull Request:

1. Create a new page and add shortcode: [bbp-search].
2. Open this page and can see the search form is appear

### Proof Screenshots or Video
https://vimeo.com/649025540/c8e0e403b8


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
